inspirehep/modules/forms/validators/simple_fields.py

```diff
@@ -88,6 +88,9 @@ def does_exist_in_inspirehep(query, collections=None):
 
 
 def duplicated_validator(property_name, property_value):
+    def _is_not_deleted(base_record, match_result):
+        return not match_result.get('_source', {}).get('deleted', False)
```

Comment:
Chained `get`s can always be replaced by `get_value`. For example, here it would be:
```python
return not get_value(match_result, '_source.deleted', default=False)
```